articles/governance/policy/assign-policy-terraform.md

```diff
@@ -38,28 +38,29 @@ for Azure Policy use the
 
 1. Create `main.tf` with the following code:
 
-   ```hcl
-   provider "azurerm" {
-       features {}
-   }
-   
-  terraform {
-    required_providers {
-      azurerm = {
-        source  = "hashicorp/azurerm"
-        version = ">= 2.96.0"
-      }
-    }
-  }
-
-  resource "azurerm_resource_policy_assignment" "auditvms" {
-    name                 =  "audit-vm-manageddisks"
-    resource_id          =  var.cust_scope
-    policy_definition_id =  "/providers/Microsoft.Authorization/policyDefinitions/06a78e20-9358-41c9-923c-fb736d382a4d"
-    description          =  "Shows all virtual machines not using managed disks"
-    display_name         =  "Audit VMs without managed disks assignment"
-   }
-   ```
+
+  ```hcl
+  provider "azurerm" {
+    features {}
+}
+
+terraform { 
+    required_providers { 
+        azurerm = { 
+            source = "hashicorp/azurerm"
+            version = ">= 2.96.0" 
+        } 
+    } 
+}
+
+resource "azurerm_resource_policy_assignment" "auditvms" { 
+    name = "audit-vm-manageddisks" 
+    resource_id = var.cust_scope 
+    policy_definition_id = "/providers/Microsoft.Authorization/policyDefinitions/06a78e20-9358-41c9-923c-fb736d382a4d" 
+    description = "Shows all virtual machines not using managed disks" 
+    display_name = "Audit VMs without managed disks assignment" 
+}
+```
```

Comment:
```suggestion

   ```hcl
   provider "azurerm" {
     features {}
 }

 terraform { 
     required_providers { 
         azurerm = { 
             source = "hashicorp/azurerm"
             version = ">= 2.96.0" 
         } 
     } 
 }

 resource "azurerm_resource_policy_assignment" "auditvms" { 
     name = "audit-vm-manageddisks" 
     resource_id = var.cust_scope 
     policy_definition_id = "/providers/Microsoft.Authorization/policyDefinitions/06a78e20-9358-41c9-923c-fb736d382a4d" 
     description = "Shows all virtual machines not using managed disks" 
     display_name = "Audit VMs without managed disks assignment" 
 }
 ```
```